src/nr_openai_observability/patchers/openai_streaming.py

```diff
@@ -34,9 +34,12 @@ def wrap_stream_generator(stream_gen):
                 trace.add_custom_attribute("completion_id", completion_id)
                 handle_start_completion(kwargs, completion_id)
                 for chunk in stream_gen:
-                    content += chunk.choices[0].delta.get("content", "")
-                    if hasattr(chunk.choices[0].delta, "role"):
-                        role = chunk.choices[0].delta.role
+                    if len(chunk.choices) > 0:
```

Comment:
I'm not sure why this happens but some chunks were returning with no `choices`. I saw that same behavior in Node I think. It might be an implementation detail or it might signal that the stream is starting?